tests/consistent-changelogs.test.ts

```diff
@@ -111,7 +111,12 @@ function readChangelogs() {
   const packagesPath = join(ROOT_PATH, 'packages');
 
   return glob
-    .sync(join(packagesPath, '*/'))
+    .sync(
+      join(
+        packagesPath,
+        '!(jest-mock-apollo|enzyme-utilities|jest-mock-router)/',
```

Comment:
These are temporary and can be removed once the empty packages are published and then removed.